app/components/DropNewPinButton.js

```diff
@@ -1,2 +1,33 @@
 
-//this should be a child of Map.js
+//this should be a child of ViewContainer
+
+import React, { Component, PropTypes, Text} from 'react-native';
+import Button from 'react-native-button';
+
+export default class DropNewPinButton extends Component {
+  constructor(props) {
+    super(props);
+    this.state= {
+     long: null,
+     lat: null,
```

Comment:
we should use 'longitude' and 'latitude' to be consistent with the navigation component we're getting the data from